R/inst/tutorials/00-helloworld/README.md

```diff
@@ -7,7 +7,10 @@ printing out 'Metaflow says: Hi!' to the terminal.**
 - Basics of Metaflow.
 - Step decorator.
 
+
 #### To play this episode:
-1. ```cd metaflow-tutorials/R/00-helloworld```
+1. ```cd tutorials/00-helloworld```
 2. ```Rscript helloworld.R show```
-3. ```Rscript helloworld.R run```
\ No newline at end of file
+3. ```Rscript helloworld.R run```
+
+In RStudio, you can run the script by directly run `source("helloworld.R")`.
```

Comment:
minor nit - _If you are using RStudio, you can run this script by directly executing `source("helloworld.R")`._

Reply:
updated according to your suggestion